### DISCUSSION TITLE
Possible to model a slung payload system?

### DISCUSSION THREAD

**Discussion by davidoort** [raisimTech/raisimLib#289] [Q&A] [answered]
I was wondering if it would be possible to model a slung payload attached to multiple quadrotors in this simulator. Something that looks like the screenshot below, with either rigid (compressible) cables or non-rigid/elastic/slack cables
![slungpayload_sys](https://user-images.githubusercontent.com/21220927/160991424-9abb369a-f623-4c23-8568-d12c0b30af98.png)

**Comment by jhwangbo** [comment, marked as answer]
Sure. You can check this doc. https://raisim.com/sections/Constraints.html
